src/RefinedAst.hs

```diff
@@ -126,30 +129,36 @@ data ReturnExp
 
 -- intermediate json output helpers ---
 instance ToJSON Behaviour where
-  toJSON (Behaviour {..}) = object  [ "name" .= _name
-                                    , "contract"  .= _contract
+  toJSON (Invariants exps contracts) = object [ "kind" .= (String "Invariants")
```

Comment:
I aded this "Kind" key to the JSON output to differentiate between `Behaviour` and `Invariants`, is this name OK for you @MrChico ?

Reply:
kind is good! 👍